### PR TITLE
Add more TV series

### PR DESCRIPTION
**Pull request by aviskarkc10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,11 @@ A list of movies/series which could have really interesting porn parody names.
 | Breaking Bad | Breaking Dad |
 | Black Hawk Down | Black Cock Down |
 | Black Widow | Blacked Widow |
+| Buffy the Vampire Slayer | Buffy the Vampire Layer |
```
Comment (mesaugat) [inline README.md]: This is not that good.

File: README.md [hunk 2]
```diff
@@ -72,6 +76,7 @@ A list of movies/series which could have really interesting porn parody names.
 | The Lion King | The Loin King |
 | The Lives of Others | The Wives of Others |
 | The Longest Day | The Longest Gay |
+| The Newsroom | The Abuse Room |
```
Comment (mesaugat) [inline README.md]: I suggest `The Dorkroom`.

File: README.md (hunk 1 above)
Comment (aviskarkc10) [reply README.md]: okay I will remove this

File: README.md (hunk 2 above)
Comment (aviskarkc10) [reply README.md]: What does `dork` mean? Can't find a relevant meaning

Comment (mesaugat) [reply README.md]: Google `define dork`.

Screen Shot 2019-11-01 at 10 12 57 AM

Comment (aviskarkc10) [reply README.md]: Ah okay. I will add that too

**Pull request merged**